src/ZfSnapPhpDebugBar/Collector/DoctrineCollectorFactory.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+namespace ZfSnapPhpDebugBar\Collector;
+
+use Interop\Container\ContainerInterface;
+use DebugBar\Bridge\DoctrineCollector;
+use Doctrine\DBAL\Logging\DebugStack;
+use Zend\ServiceManager\FactoryInterface;
+use Zend\ServiceManager\ServiceLocatorInterface;
+
+class DoctrineCollectorFactory implements FactoryInterface
+{
+    /**
+     * {@inheritdoc}
+     */
+    public function __invoke(ContainerInterface $container, $requestedName, array $options = null)
+    {
+        $debugStack = new DebugStack;
+        $container->get('doctrine.configuration.orm_default')->setSQLLogger($debugStack);
```

Comment:
I think we can do it better. For example using delegators with configuration object.